cmd/kyma/alpha/create/module/moduleconfig.go

```diff
@@ -13,19 +13,19 @@ import (
 )
 
 type Config struct {
-	Name              string                     `yaml:"name"`             //required, the name of the Module
-	Version           string                     `yaml:"version"`          //required, the version of the Module
-	Channel           string                     `yaml:"channel"`          //required, channel that should be used in the ModuleTemplate
-	ManifestPath      string                     `yaml:"manifest"`         //required, reference to the manifests, must be a relative file name.
-	DefaultCRPath     string                     `yaml:"defaultCR"`        //optional, reference to a YAML file containing the default CR for the module, must be a relative file name.
-	ResourceName      string                     `yaml:"resourceName"`     //optional, default={NAME}-{CHANNEL}, the name for the ModuleTemplate that will be created
-	Namespace         string                     `yaml:"namespace"`        //optional, default=kcp-system, the namespace where the ModuleTemplate will be deployed
-	Security          string                     `yaml:"security"`         //optional, name of the security scanners config file
-	Internal          bool                       `yaml:"internal"`         //optional, default=false, determines whether the ModuleTemplate should have the internal flag or not
-	Beta              bool                       `yaml:"beta"`             //optional, default=false, determines whether the ModuleTemplate should have the beta flag or not
-	Labels            map[string]string          `yaml:"labels"`           //optional, additional labels for the ModuleTemplate
-	Annotations       map[string]string          `yaml:"annotations"`      //optional, additional annotations for the ModuleTemplate
-	CustomStateChecks []v1beta2.CustomStateCheck `yaml:"customStateCheck"` //optional, specifies custom state check for module
+	Name              string                     `yaml:"name" comment:"required, the name of the Module"`
```

Comment:
What is the purpose of that change? The `comment` tag is not documented in the [yaml package API](https://pkg.go.dev/gopkg.in/yaml.v3#Marshal)